tests/baselines/reference/parserArrowFunctionExpression10.js

```diff
@@ -1,8 +0,0 @@
-//// [parserArrowFunctionExpression10.ts]

```

Comment:
Ugh, this is a rename, not a deletion.

Reply:
Yeah, that's why I wanted to have my first commit be only test changes, so the following commits were clear as to what changed.